packages/paste-website/src/pages/components/anchor/index.mdx

```diff
@@ -188,17 +198,18 @@ const Component = () => <Anchor href="https://paste.twilio.design">Go to Paste</
-| onFocus?      | `(event: React.FocusEvent<HTMLElement>)` |                                                                                                         | null      |
-| onBlur?       | `(event: React.FocusEvent<HTMLElement>)` |                                                                                                         | null      |
+| Prop                  | Type                                     | Description                                                                                             | Default                                |
+| --------------------- | ---------------------------------------- | ------------------------------------------------------------------------------------------------------- | -------------------------------------- |
+| href?                 | string                                   | A URL to route to. Required.                                                                            | null                                   |
+| rel?                  | string                                   | Sets the anchor rel attribute. If external href, defaults to 'noreferrer noopener'. Can be overwritten. | false                                  |
+| showExternal?         | boolean                                  | Shows the link external icon.                                                                           | false                                  |
+| tabindex?             | AnchorTabIndexes                         | '0', '-1'                                                                                               | null                                   |
+| target?               | AnchorTargets                            | '\_self', '\_blank', '\_parent', '\_top'. If external href, defaults to '\_blank'. Can be overwritten.  | null                                   |
+| variant?              | AnchorVariants                           | 'default', 'inverse'                                                                                    | 'default'                              |
+| onClick?              | `(event: React.MouseEvent<HTMLElement>)` |                                                                                                         | null                                   |
+| onFocus?              | `(event: React.FocusEvent<HTMLElement>)` |                                                                                                         | null                                   |
+| onBlur?               | `(event: React.FocusEvent<HTMLElement>)` |                                                                                                         | null                                   |
+| i18nExternalLinkLabel | string                                   | Title for `showExternal` icon                                                                           | '(link takes you to an external page)' |
```

Comment:
this wraps in the table on the website... not super cute but how do I fix

Reply:
I'm tweaking the tables in the table PR. I wonder if that will resolve your concern?

I remove layout fixed from the props tables, so the columns are more sized by content than arbitrarily just being equal widths

Reply:
ok awesome, it only wrapped by 1 letter on my comp so hopefully that'll do